.env.example

```diff
@@ -51,3 +51,6 @@ DATABASE_ADAPTER=postgres
 # any value https://github.com/vercel/ms can accept
 USER_ACTIVITY_YIELD_INTERVAL=1h
 USER_ACTIVITY_TTL=10min
+
+# User import (csv or xlsx are supported)
+USER_IMPORT_TEMPLATE_FORMAT=
```

Comment:
Since other entities could potentially be imported, why adding `USER` prefix?
`IMPORT_TEMPLATE_FORMAT` would be more generic & enough.
This makes sense in case other entities are imported with a different format. 
Is there such a case? @zcuric

Reply:
@malizeus This is related to the current code base based on master. You can do whatever you please.